modules/juce_audio_plugin_client/VST/juce_VST_Wrapper.cpp

```diff
@@ -2109,6 +2116,7 @@ class JuceVSTWrapper  : public AudioProcessorListener,
     double sampleRate = 44100.0;
     int32 blockSize = 1024;
     Vst2::AEffect vstEffect;
+    std::atomic_int lastChangeFlags {0};

```

Comment:
`atomic<int>` is more canonical (unless you see JUCE style prefers these)

Reply:
ok. will do. (didn't know it's that opinionated :) )